worldengine/a_star.py

```diff
@@ -36,42 +34,45 @@ def __eq__( self, n ):
 
 
 class AStar:
+    def __init__(self, map_handler):
+        self.mh = map_handler
+        self.o = []
```

Comment:
In theory, most people shouldn't ever have to look here. However I fixed up a lot of this class to be more readable. There is still work to be done. :)